LayoutTests/imported/w3c/web-platform-tests/svg/idlharness.window.js

```diff
@@ -28,41 +28,24 @@ const elements = [
   'polygon',
   'text',
   'tspan',
-  'tref',
   'textPath',
-  'altGlyph',
-  'altGlyphDef',
-  'altGlyphItem',
-  'glyphRef',
   'marker',
-  'colorProfile',
   'linearGradient',
   'radialGradient',
   'meshGradient',
   'stop',
   'pattern',
   'clipPath',
   'mask',
-  'cursor',
   'a',
   'view',
   'script',
   'animate',
   'set',
+  'discard',
```

Comment:
Please remove this line.

Reply:
I synced it from WPT - if we need to delete, we should do it upstream and if I delete it from local copy, upon next WPT sync, it will be added back.

Reply:
So what is the purpose of this PR then?

1. We can't add `onrepeat` because we have to implement `repeatEvent` first
2. We can't add `onend` because it exposes any bug that we need to fix first
3. We can't assume `<discard>` will be considered because Blink has dropped it and Gecko has the intent to prototype
4. We can't remove `<discard>` from idlharness.window.js because we will have to upstream it.

Reply:
It is to add `onbegin`, which don't add yet.

Reply:
Yes but you are introducing many other failures which are not clear to me if they are real failures or not. For me the _expected.txt_ file is the truth. So when I see a failure like 
```
FAIL SVGDiscardElement interface: existence and properties of interface object assert_own_property: self does not have own property "SVGDiscardElement" expected property "SVGDiscardElement" missing
```
It tells me that we have a bug by not implementing `SVGDiscardElement` which is not true.

So if we add this line to the _expected.txt_ file, it will not be the truth for me anymore.

Reply:
You can add `onbegin` without syncing the WPT.

Reply:
OK, I will remove updated test. :-)